iOSDemo/ViewControllers/DemoViewController.swift

```diff
@@ -3,10 +3,18 @@ import UIKit
 import SweetUIKit
 
 enum DemoItem: Int {
-    case edit, collection, searchableCollection
+    case
+        edit,
+        collection,
+        searchableCollection
 
     static var titles: [String] {
-        let allItems: [DemoItem] = [.edit, .collection, .searchableCollection]
+        let allItems: [DemoItem] = [
+            .edit,
+            .collection,
+            .searchableCollection,
```

Comment:
Same comment about code formatting here. Any arguments for having it multiline instead of one-line?

Reply:
i actually kind of prefer this since you can more clearly see what's been added and removed in diffs, but if you'd prefer I put it back, lmk.

Reply:
I think having them as oneliners would make things consistent :)

Reply:
Will do 👍